plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryThumbnails.php

```diff
@@ -75,50 +75,41 @@ protected function render( $attributes, $content, $block ) {
 			return '';
 		}
 
-		// Will eventually be replaced by a slider. Temporary solution.
-		$default_number_of_thumbnails = 3;
-		$number_of_thumbnails         = isset( $attributes['numberOfThumbnails'] ) && is_numeric( $attributes['numberOfThumbnails'] ) ? $attributes['numberOfThumbnails'] : $default_number_of_thumbnails;
-		$number_of_images             = count( $product_gallery_images );
-		// If the number of thumbnails is greater than the number of images, set the number of thumbnails to the number of images.
-		// But not less than than 3 (default number of thumbnails).
-		$thumbnails_layout          = max( min( $number_of_images, $number_of_thumbnails ), $default_number_of_thumbnails );
-		$number_of_thumbnails_class = 'wc-block-product-gallery-thumbnails--number-of-thumbnails-' . $thumbnails_layout;
-		$remaining_thumbnails_count = $number_of_images - $number_of_thumbnails;
-		wp_interactivity_config( 'woocommerce/product-gallery', array( 'numberOfThumbnails' => $number_of_thumbnails ) );
-		// End of temporary solution.
+		$thumbnail_size   = str_replace( '%', '', $attributes['thumbnailSize'] ?? '20%' );
```

Comment:
Similar to https://github.com/woocommerce/woocommerce/pull/56069/files#r1995705931
```suggestion
		$thumbnail_size   = str_replace( '%', '', $attributes['thumbnailSize'] ?? '33%' );
```